app/assets/javascripts/modules/collapsible_collection.js

```diff
@@ -98,28 +98,26 @@
 
   CollapsibleCollection.prototype.closeAll = function closeAll(event){
     for (var section in this.collapsibles) {
+      var $button = $(this.collapsibles[section].$clickTarget[0].firstChild)
```

Comment:
Does this work? Following the logic from line 25, where `this.closeAll()` is called with no 'event' in the object constructor (rather than in response to a click) I'd expect a `Cannot get firstChild of undefined` error here 🤔

Reply:
yeah, it works as the entire collapse block is passed through. i've removed the leftover event as i've already removed the preventDefault

Reply:
also replaced with querySelector as we have the parent h2 available as the $clickTarget